src/redux/club/__tests__/auth.js

```diff
@@ -0,0 +1,36 @@
+import {
+  LOG_OUT, logOut,
+  AUTH, auth,
+} from '../auth'
+
+import reducer from '../auth'
+import initialState from 'vclub/redux/initialClubState'
+
+// action creator
+test('logOut action creator creates proper action', () => {
+  const nextRoom = 'CHAT'
+  const expectedAction = {
+    type: LOG_OUT,
+    meta: {
+      sideEffect: () => {}
+    },
+  }
+  const action = logOut()
+  expect(action.type).toEqual(expectedAction.type)
+
+  const ioSocket = {
+    emit: jest.fn(),
+  };
+
+  action.meta.sideEffect({ ioSocket })
```

Comment:
1. Check action.meta.sideEffect
2. Use another test case for side effect execution test